Mathlib/AlgebraicTopology/Quasicategory/Nerve.lean

```diff
@@ -26,6 +26,6 @@ namespace CategoryTheory.Nerve
 /-- By virtue of satisfying the `StrictSegal` condition, the nerve of a
 category is a `Quasicategory`. -/
 instance quasicategory {C : Type u} [Category.{v} C] :
-  Quasicategory (nerve C) := inferInstance
+  Quasicategory (nerve C) := strictSegal C |>.quasicategory
```

Comment:
If this is not found by typeclass inference, there is a design problem.

Reply:
Admittedly, I don't understand typeclass inference in Lean very well, but doesn't the fact that `StrictSegal` is now a structure rather than a class preclude this from being inferred?

Reply:
For any theorem which proves that a property (in a class) when there is a structure `StrictSegal X`, you should add an instance under the assumption `IsStrictSegal X`.

Reply:
Thanks, I believe I have resolved this in commits [eb69084](https://github.com/leanprover-community/mathlib4/pull/20668/commits/eb69084a9ab667918112c0300dd0478814f792a2) and [b047adf](https://github.com/leanprover-community/mathlib4/pull/20668/commits/b047adfd43b9b17bf33a2689385917c6f525ccb9).